quickstart/201-machine-learning-private/network.tf

```diff
@@ -0,0 +1,89 @@
+# Virtual Network
+resource "azurerm_virtual_network" "default" {
+  name                = "${var.name}-${var.environment}-vnet"
```

Comment:
This should align with Azure naming conventions mentioned above and be a prefix instead.